In [1]:
import numpy as np

__Задача 1__

О непрерывной равномерно распределённой случайной величине `B` известно, что её дисперсия равна 0.2. Можно ли найти правую границу величины `B` и её среднее значение зная, что левая граница равна 0.5? Если да, найдите их.

_Нам известно, что в равномерно распределенной величине $D(B) = \dfrac{(b - a)^2}{12}$, а ее среднее значение $M(B) = \dfrac{a + b}{2}$_ 

_$(b - a)^2 = 12D(B)$ при $D(B)=0.2$ и $a=0.5$_

_$b^2 - b + 0.25 = 2.4$_

_$b = \dfrac{1 \pm \sqrt{1 + 4 \cdot 2.15}}{2}$ при $b > 0.5$ т.к. мы ищем правую границу_

In [2]:
b = (1 + np.sqrt(1 + 4*2.15))/2
b

2.049193338482967

In [3]:
a = 0.5
m_b = (a + b)/2
m_b

1.2745966692414834

_$b \approx 2.04919$_

_$M(B) \approx 1.2746$_

__Задача 2__

На сколько сигм (средних квадратических отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой `M(X) = 178` и `D(X) = 25`?

_зная, что рост людей соответствует нормальному распределению, значит $D(X)=\sigma^2$_

_значит $\sigma^2 = \sqrt{D(X)}$_

In [4]:
sig = np.sqrt(25)
diff = (190 - 178) / sig
diff

2.4

_$отклонение = 2.4 \sigma$_

__Задача 3__

Рост взрослого населения города `X` имеет нормальное распределение, причём, средний рост равен 174 см, а среднее квадратическое отклонение равно 8 см. С помощью правил стандартного отклонения, двух и трёх сигм посчитайте, какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

1. больше 182 см?
2. больше 190 см?
3. от 166 см до 190 см?
4. от 166 см до 182 см?
5. от 158 см до 190 см?
6. не выше 150 см или не ниже 190 см?
7. не выше 150 см или не ниже 198 см?
8. ниже 166 см?


_зная, что:_

* От $a - \sigma$ до $a + \sigma$ содержит около $68\%$ вероятностной массы
* От $a - 2\sigma$ до $a + 2\sigma$ — около $95\%$ массы
* От $a - 3\sigma$ до $a + 3\sigma$ — около $99.7\%$ массы

_$M(X) = 174 см.$ и при $\sigma = 8 см.$_

_$P(X \in [166, 182]) = 0.68$_

_$P(X \in [158, 190]) = 0.95$_

_$P(X \in [150, 198]) = 0.997$_

_1. больше 182 см?_

_$P(X > 182) = \dfrac{1 - 0.68}{2} = 0.16$_

_2. больше 190 см?_

_$P(X > 190) = 0.025$_

_3. от 166 см до 190 см?_

_$P(X \in [166, 190]) = P(X \in [166, 182]) + \dfrac{P(X \in [158, 190]) - P(X \in [166, 182]}{2} = 0.815$_

_4. от 166 см до 182 см?_

_$P(X \in [166, 182]) = 0.68$_

_5. от 158 см до 190 см?_

_$P(X \in [158, 190]) = 0.95$_

_6. не выше 150 см или не ниже 190 см?_

_$\overline{P}(X \in [150, 190]) = \dfrac{P(X \in [150, 198]) - P(X \in [158, 190])}{2} + \overline{P}(X \in [150, 198]) = 0.0265$_

_7. не выше 150 см или не ниже 198 см?_

_$\overline{P}(X \in [150, 198]) = 0.003$_

_8. ниже 166 см?_

_$P(X < 166) = \dfrac{\overline{P}(X \in [166, 182])}{2} = 0.16$_

__Задача 4__ (на подумать)

Каким образом связаны между собой понятие квантиля и правила двух и трёх сигм? 

Данное задание - на подумать, оформлять его не обязательно. Этот вопрос мы обсудим в начале следующего занятия.

_Правило $двух / трех$ сигм говорит нам о том, что $68\% / 95\%$ выборки находится в интервалах $(a - 2\sigma, a + 2\sigma) / (a - 3\sigma, a + 3\sigma)$_

_другими словами квантиль порядка $Q(\dfrac{1 - 0.68}{2}) = Q(0.16) = a - 2\sigma$_

_$Q(1 - \dfrac{1 - 0.68}{2}) = Q(0.84) = a + 2\sigma$_

_$Q(\dfrac{1 - 0.95}{2}) = Q(0.025) = a - 3\sigma$_

_$Q(1 - \dfrac{1 - 0.95}{2}) = Q(0.975) = a + 3\sigma$_

_где $a$ - выборочное среднее, а $\sigma$ - среднеквадратическое отклонение_